app/models/lettings_log.rb

```diff
@@ -521,6 +521,7 @@ def is_carehome?
 
 private
 
+  # show in diff
```

Comment:
Could you remove these comments please?

Reply:
yes I definitely will, they're just so this method is visible in the diff as I wanted to discuss it as part of the review